pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -47,6 +47,22 @@ let
         };
       };
 
+      _2gua.rainbow-brackets = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          publisher = "2gua";
+          name = "Rainbow Brackets";
```

Comment:
`name` should have been `rainbow-brackets`.

Reply:
done